Frontend/src/components/TableView/TableView.svelte

```diff
@@ -74,7 +78,7 @@
   };
 
   let actualRowsFittingOnPage = false;
-  let rowsPerPage = maxRowsThatMightFitOnPage();
+  let rowsPerPage = 25; // maxRowsThatMightFitOnPage();
```

Comment:
I didn't test it, but I guess if you hardcode this to 25 it won't look good on different screen sizes or if you resize the browser window?

Reply:
Yes, not optimal, but on very large screens you simply have a bit of empty space and on smaller screens you can scroll. Fixed page size is not unusual and the software will anyways only be used on our two laptops.

Reply:
Ok. I think you can remove the old code then instead of using comments.